### PR TITLE
update device_profiles.md

### PR DESCRIPTION
**Pull request by EThom73**
This now includes the modified wording around standard builds form the IMS overview document. 

The items under  "# Related Standards, Policies and Processes" are unlinked as the new IMS  repo and relevant updated documents do not exist yet. They will be linked in the future.

### CODE REVIEW

**Comment by chrisblackburn** [comment]
I think this may need updating:

Because of our heavy use of cloud-based software, we do not believe removing default admin access will prevent people from working productively. If a team member can provide a business case for why they need to retain admin access to their device as default, the ISMS governance team will review this policy.

to something like:

Team members will have access to an admin account for the purpose of upgrading system software. This account must not be logged in to directly, and should be used only for the purpose of updating installed software or installing pre-approved software.

**Comment by chrisblackburn** [comment]
+ 

Members of the technology team may use an admin account during day-to-day work if using a less privileged account significantly impedes their productivity

**Comment by EThom73** [comment]
> Members of the technology team may use an admin account during day-to-day work if using a less privileged account significantly impedes their productivity

Not sure where this should be going . Do you mean instead of the section on Engineering and Delivery teams?

**Comment by chrisblackburn** [comment]
Oh whoops, I didn't realise this was in there already. I was just looking at the diff. Please ignore

File: guides/security/device_profiles.md [hunk 1]
```diff
@@ -1,8 +1,10 @@
-# Admin profiles and standard user profiles on devices
+# Device profiles and standard builds
 
-As part of our commitment to secure working practices, we install separate admin and standard user profiles on laptops. The reason this restriction is in place is to prevent the accidental downloading and installation of malicious software on company devices, to close an attack vector, and to protect both Made Tech and our customers.
+Made Tech uses standard builds for the devices issued to team members to facilitate support and maintenance, and enforce company standards in security. Standard builds are not used where the IMS management team and/or leadership have agreed there is a clear case for not doing so, as well as when the application or purpose of platform requires alternative or special configurations.
```
[the remark this answers is not in the document]
Comment (EThom73) [reply guides/security/device_profiles.md]: It's been updated https://github.com/madetech/handbook/pull/249

IMS Management is currently the standard language we and our consultant are using. 

I am also the IMS Manager which is one manag- too many, but we're just rolling with it until we've settled everything. Don't want to generate a gardening task of changing a common term just yet, though open to it in the future.

File: guides/security/device_profiles.md [hunk 2]
```diff
@@ -1,8 +1,10 @@
...
 
-Because of our heavy use of cloud-based software, we do not believe removing default admin access will prevent people from working productively. If a team member can provide a business case for why they need to retain admin access to their device as default, the ISMS governance team will review this policy.
+As part of our commitment to secure working practices, we install separate admin and user profiles on laptops. The reason this restriction is in place is to prevent the accidental downloading and installation of malicious software on company devices and to protect both Made Tech and our customers.
```
[the remark this answers is not in the document]
Comment (EThom73) [reply guides/security/device_profiles.md]: This point is expanded on in other policies that will be linked out (eventually - currently sitting as gists)

File: guides/security/device_profiles.md (hunk 1 above)
Comment (lukemorton) [inline guides/security/device_profiles.md]: If we are using acronyms like "IMS team" we should link to an explanation of these terms as they aren't plain english. The audience of our Handbook includes potential employees and customers, as well as current staff.

Comment (EThom73) [reply guides/security/device_profiles.md]: The README of the IMS Section includes a definition of IMS in the first paragraph. 

Do we need to repeat this definition in every file?

File: guides/security/device_profiles.md [hunk 3]
```diff
@@ -29,3 +31,9 @@ We have agreed a clear business case for people in engineering roles to use an a
  - Evernote
 
  For Mac users, any app downloaded from the App Store is considered safe to install on your device.
+ 
+ # Related Standards, Policies and Processes
+ - AUP Policy
```
Comment (danielburnley) [inline guides/security/device_profiles.md]: Not sure if these are meant to be links or just words - fine either way just wanted to draw attention if it wasn't intentional

**Pull request merged**